src/test/java/se/sundsvall/snailmail/api/SnailMailResourceTest.java

```diff
@@ -1,39 +1,100 @@
 package se.sundsvall.snailmail.api;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.groups.Tuple.tuple;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
-import static se.sundsvall.snailmail.TestDataFactory.buildSendSnailMailRequest;
```

Comment:
Remove `buildSendSnailMailRequest`-method in `TestDataFactory` (if it is not used elsewhere than here)?

Reply:
Its used elsewhere.